src/main/java/TaskList.java

```diff
@@ -0,0 +1,40 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    String indent = "    ";
+
+    public void addTask(Task task) {
+        taskList.add(task);
+    }
+
+    public String showNewest() {
+        int length = taskList.size();
+        Task task = taskList.get(length - 1);
+        return task.getName();
+    }
+
+    public String showList() {
+        Task task;
+        String status;
+        String finalOutput = new String("");
+        
+        for (int i = 0; i < taskList.size(); i++) {
+            task = taskList.get(i);
+            status = task.checkStatus();
+            finalOutput = finalOutput + indent + Integer.toString(i + 1) + "." + status + "\n";
+        }
+        return finalOutput;
```

Comment:
String concatenation can be quite expensive, maybe you can consider trying using StringBuilder? Other than that clean and readable code 👍